Libraries/Image/Image.android.js

```diff
@@ -109,6 +110,14 @@ var Image = React.createClass({
 
   statics: {
     resizeMode: ImageResizeMode,
+
+    /**
+    * prefetch image to later use
```

Comment:
nit: for later use. Start with a capital letter. Add more docs to explain how this works. Where are the images stored? Is there any limit on the memory this uses (at runtime, on the SD card)?